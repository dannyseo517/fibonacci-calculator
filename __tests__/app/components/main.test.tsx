import React from 'react';
import { shallow } from 'enzyme';
import Main from 'Components/main';

describe('Main component', () => {
    it('should render initial layout', () => {
        const component = shallow(<Main />);
        expect(component.getElements()).toMatchSnapshot();
    });
});

