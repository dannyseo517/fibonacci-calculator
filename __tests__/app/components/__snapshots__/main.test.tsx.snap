// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component should render initial layout 1`] = `
Array [
  <div
    className="main-container"
  >
    <img
      className="heading-img"
      height={198.7}
      src={Object {}}
      width={300}
    />
    <div
      className="heading-text"
    >
      <span>
        Fibonacci 
      </span>
      <span>
        Calculator
      </span>
    </div>
    <div
      className="heading-subtext"
    >
      by Danny Seo
    </div>
    <Connect(FibonacciInput) />
    <Connect(Results) />
  </div>,
]
`;
